src/bundle/Resources/public/scss/_typography.scss

```diff
@@ -0,0 +1,37 @@
+h1 {
+    font-size: 1.75rem;
+}
+
+h2 {
+    font-size: 1.5rem;
+}
+
+h3 {
+    font-size: 1.375rem;
+}
+
+h4 {
+    font-size: 1.25rem;
+}
+
+h5 {
+    font-size: 1.125rem;
+}
+
+.ez-dashboard .ez-main-row {
+    h4 {
+        font-weight: bold;
+    }
+}
+
+.ez-page-title {
+    h1 {
+        font-weight: bold;
+    }
+}
+
+.ez-content-edit {
+    h1 {
```

Comment:
I think the selector is too generic it will have an impact on every `h1` in content edit. Why only `h1` and only in content edit (and some other places)?
Maybe it's good idea to set it by boostrap variable for every `hx` in the app? 
```
$headings-font-weight:   500 !default;
```